components/AssetFormSellPopupMenu/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import {
+  MenuContainer,
+  PopupMenuButton,
+  Menu,
+  MenuItem,
+} from './AssetFormSellPopupMenu.styled';
+import { ReactComponent as Ellipsis } from '../../public/ellipsis.svg';
+import { useModalContext, MODAL_TYPES } from '../Provider';
+
+const AssetFormSellPopupMenu = (): JSX.Element => {
```

Comment:
The pop up isn't disappearing when I click outside of it. Would like that added since I think it's expected behavior.